content/docs/start/index.md

```diff
@@ -1,3 +1,9 @@
+---
+description: 'A step-by-step introduction: Learn how to use DVC to organize,
+version, and efficiently reproduce your data science or machine learning
+project.'
```

Comment:
Not bad but sounds a lot like the desc of the Versioning tutorial. Should we try to make them clearly different? The GS is basically a primer on all of DVC's feature layers (there's 3-4) and the Versioning tutorial is about that one layer specifically (so, closer to the Data Versioning page of the Get Started, but deeper and with a real repo example).

Reply:
In fact notice how the current autogenerated desc Google shows comes from the first bullet in the doc, which refers to Data Versioning only. Let's try to include all 4 bullets in the SEO desc. please, unless you have a better idea.

Reply:
OK, good idea. Reworked this to include all 4 bullets.